src/DynamoCore/Scheduler/UpdateRenderPackageAsyncTask.cs

```diff
@@ -151,9 +147,15 @@ private void GetRenderPackagesFromMirrorData(MirrorData mirrorData, bool display
 
             if (mirrorData.IsCollection)
             {
+                int count = 0;
                 foreach (var el in mirrorData.GetElements())
                 {
-                    GetRenderPackagesFromMirrorData(el, displayLabels, isNodeSelected, ref labelMap, ref count);
+                    if (el.IsCollection || el.Data is IGraphicItem)
```

Comment:
`IsCollection` is defined as `return svData.IsArray`; I can't think of a case where it is `IEnumerable`.